apps/astarte_realm_management/lib/astarte_realm_management/config.ex

```diff
@@ -42,11 +42,5 @@ defmodule Astarte.RealmManagement.Config do
   @doc """
   Returns Cassandra nodes formatted in the Xandra format.
   """
-  defdelegate xandra_nodes, to: DataAccessConfig
-  defdelegate xandra_nodes!, to: DataAccessConfig
-
   defdelegate xandra_options!, to: DataAccessConfig
-
-  defdelegate astarte_instance_id!, to: DataAccessConfig
-  defdelegate astarte_instance_id, to: DataAccessConfig
```

Comment:
As agreed upon IRL, let's make really sure that this is not called anymore before removing it

Reply:
Actually upon further inverstigation this does not seem to happen :+1: 

Apparently `astarte_instance_id` is only called from inside `DataAccess`, `RealmManagement` does not call it directly